test/test_unistd.rs

```diff
@@ -64,22 +64,27 @@ fn test_wait() {
 
 #[test]
 fn test_mkstemp() {
-    let result = mkstemp("/tmp/nix_tempfile.XXXXXX");
+    use std::env;
```

Comment:
stylistically, I prefer to put all `use` statements at the top of the file, unless there's a good reason not to.